src/gui/src/dbDescriptors.cpp

```diff
@@ -1639,4 +1648,139 @@ bool DbTechLayerDescriptor::getAllObjects(SelectionSet& objects) const
   return true;
 }
 
+//////////////////////////////////////////////////
+
+static const char*
+getAccessTypeName(odb::dbAccessPoint::AccessType type)
+{
+  switch(type) {
+  case odb::dbAccessPoint::OnGrid:
+    return "OnGrid";
+  case odb::dbAccessPoint::HalfGrid:
+    return "HalfGrid";
+  case odb::dbAccessPoint::Center:
+    return "Center";
+  case odb::dbAccessPoint::EncOpt:
+    return "EncOpt";
+  case odb::dbAccessPoint::NearbyGrid:
+    return "NearbyGrid";
+  }
+  return "Unknown";
+}
+
```

Comment:
Would this be better in ODB so all of OpenROAD can share this nomenclature?

Reply:
I'll change it.